content/docs/2.0/scalers/azure-log-analytics.md

```diff
@@ -0,0 +1,150 @@
++++
+title = "Azure Log Analytics"
+layout = "scaler"
+availability = "v2.0+"
+maintainer = "Community"
+description = "Scale applications based on Azure Log Analytics query result"
+go_file = "azure_log_analytics_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `azure-log-analytics` trigger for Azure Log Analytics query result (All auth parameters in metadata is not real, generated randomly).
+
+```yaml
+triggers:
+  - type: azure-log-analytics
+    metadata:

```

Comment:
Adding @ahmelsayed for input on the ForEnv renaming we are doing

Reply:
@tomkerkhove , I have added support for 'FromEnv' in last commit for scaler.
CC: @ahmelsayed

Reply:
Thanks!

Reply:
Can we update the docs here to list those new properties as well please?

Reply:
@tomkerkhove , I have added description of 'FromEnv' in last commit.